src/core/core.element.js

```diff
@@ -63,8 +63,13 @@ class Element {
 		this.hidden = false;
 	}
 
-	pivot() {
+	pivot(animationsDisabled) {
 		var me = this;
+		if (animationsDisabled) {
+			me._view = me._model;
```

Comment:
I think this works, but would probably better belong in `transition`. My understanding is that the main point of `pivot` is to reset `_start` and that the point of `transition` is to set `_view` (we should probably document the functions)

`pivot` and `transition` both have a big of junk in them right now that are mainly cause by `Tooltip` not properly initializing these variables in the way that other elements do. I looked a little bit at trying to untangle some of it earlier, but didn't have the time. I don't think `pivot` should be initializing `_view` (why does it only do it once? what about subsequent updates) and I don't think transition should have to initialize `_view` and `_start` (`_view` should probably be initialized in a constructor and `_start` is already set in `pivot`, so we should make sure `Tooltip` calls `pivot` like everything else does)

Reply:
I'm planning to take a deeper look at improving the animation system, in the near future (when I have enough time). But it will be another PR, if I ever get it done.

Reply:
Yeah, I wouldn't try to make any of the other changes in this PR. But maybe we should at least pass the `animationsDisabled` flag to `transition` in this PR so that we don't drift further from the original intentions of the methods?

Reply:
Ok, so how about breaking it a little then? :)
`pivot` should work the same, animations disabled or not.
`transition` is not called when animations are disabled (for dataset elements, for now)